pkg/cmd/util.go

```diff
@@ -24,6 +24,8 @@ func AddCommonFlags(c *cobra.Command, cfg *api.Config) {
 		"DEPRECATED: Always pull the builder image even if it is present locally")
 	c.Flags().VarP(&(cfg.BuilderPullPolicy), "pull-policy", "p",
 		"Specify when to pull the builder image (always, never or if-not-present)")
+	c.Flags().Var(&(cfg.PreviousImagePullPolicy), "incremental-pull-policy",
+		"Specify when to pull the previous image for incremental builds (always, never or if-not-present)")
```

Comment:
A little explanation why the two different pull policy are needed:
1. for OpenShift we want to make sure we always use the latest built image, the actual build may land on different nodes, which may or may not have the most recent image, thus the `always` policy assures we do have the latest, which will be in the output image stream
2. for CLI usecases, `pull-if-not-present` is the optimal, because when I'm developing locally I'd like to re-use already downloaded deps. @mfojtik gave here a really good example of a conference wifi, when I'm trying to rebuild the image (eg. I fixed a slight type just before the presentation) but without downloading half of the internet (yeah I'm looking at you java/mvn guys) but rather reusing what I have in my previous image.